spec/std/http/cookie_spec.cr

```diff
@@ -146,14 +146,34 @@ module HTTP
 
       it "parse max-age as seconds from Time.now" do
         cookie = parse_set_cookie("a=1; max-age=10")
-        delta = cookie.expires.not_nil! - Time.now
-        delta.should be > 9.seconds
-        delta.should be < 11.seconds
+        cookie.max_age.should eq 10.seconds
 
         cookie = parse_set_cookie("a=1; max-age=0")
-        delta = Time.now - cookie.expires.not_nil!
-        delta.should be > 0.seconds
-        delta.should be < 1.seconds
+        cookie.max_age.should eq 0.seconds
+      end
+    end
+
+    describe "expiration_time" do
+      it "by max-age=0" do
+        cookie = parse_set_cookie("bla=1; max-age=0")
+        (cookie.expiration_time.not_nil! <= (Time.now + 1.seconds)).should eq true
```

Comment:
`eq true` should be written as `be_true` in all specs.
And the tested value is probably better understandable as `Time.now - cookie.expiration_time <= 1.seconds`